From 9611f5ffc30ece6bbab86f53b594cec7a9ea2916 Mon Sep 17 00:00:00 2001
From: Lawrence Rust <lvr@softsystem.co.uk>
Date: Thu, 20 Jun 2013 11:40:31 +0100
Subject: [PATCH 017/227] AFD: Increase default bitrate to 5Mbps for H264
 streams

ffmpeg doesn't provide a bitrate for some H264 streams so it's defaulted
at present to 500kbps.  The RingBuffer uses this value to setup initial read
block sizes.  With HDTV this value is too low which causes RingBuffer to
underrun initially.  So increase it to a more reasonable value.

Signed-off-by: Lawrence Rust <lvr@softsystem.co.uk>
---
 mythtv/libs/libmythtv/avformatdecoder.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythtv/libs/libmythtv/avformatdecoder.cpp b/mythtv/libs/libmythtv/avformatdecoder.cpp
index 7ae0ce2..27d72d2 100644
--- a/mythtv/libs/libmythtv/avformatdecoder.cpp
+++ b/mythtv/libs/libmythtv/avformatdecoder.cpp
@@ -1852,7 +1852,7 @@ int AvFormatDecoder::ScanStreams(bool novideo)
                 // use optimisations for low bitrate (audio and data) streams.
                 if (enc->bit_rate == 0)
                 {
-                    enc->bit_rate = 500000;
+                    enc->bit_rate = 8000000; // Typical for HD
                     unknownbitrate = true;
                 }
 
-- 
1.7.9.5

